tests/test_gui.py

```diff
@@ -0,0 +1,1152 @@
+# Copyright (c) 2019 Shotgun Software Inc.
+#
+# CONFIDENTIAL AND PROPRIETARY
+#
+# This work is provided "AS IS" and subject to the Shotgun Pipeline Toolkit
+# Source Code License included in this distribution package. See LICENSE.
+# By accessing, using, copying or modifying this work you indicate your
+# agreement to the Shotgun Pipeline Toolkit Source Code License. All rights
+# not expressly granted therein are reserved by Shotgun Software Inc.
+
+import pytest
+import subprocess
+import time
+import os
+import sys
+import sgtk
+from tk_toolchain.cmd_line_tools import tk_run_app
+from MA.UI import first
```

Comment:
You need to put this inside the import so the test can be skipped properly is MA is missing.